backend/routes/stocks.js

```diff
@@ -1,29 +1,38 @@
 import express from "express";
 import { Stock } from "../models/stock.js";
+import { Follow } from "../models/follow.js";
 
 const router = express.Router();
 
 // Create a new stock in database
 router.post("/stocks", async (req, res) => {
     try {
         const stock = await Stock.create(req.body);
-        res.json(stock);
+        res.json({ stock });
     } catch (error) {
-        res.send(error);
+        res.send({ error });
     }
 });
 
 // Get a specific stock from database
 router.get("/stocks/:ticker", async (req, res) => {
     const ticker = req.params.ticker.toUpperCase();
-    try {
-        const stock = await Stock.findOne({
-            where: { ticker }
-        });
-        res.json(stock);
-    } catch (error) {
-        res.send(error);
-    }
+    const stock = await Stock.findOne({
+        where: { ticker }
+    });
+
+    const user = req.session.user;
+
+    const StockId = stock.id;
+    const UserId = user.id;
+
+    const following = await Follow.findOne({ where: { UserId, StockId } });
+
+    following
+        ? (stock.dataValues.following = true)
+        : (stock.dataValues.following = false);
```

Comment:
Did you have to change the model to get this dataValues? How does it work?

Reply:
Sequelize automatically adds that dataValues prop, but when you send out stock as a response, it automatically removes the dataValues prop. I ran into a bug trying to access the props in stock and that was the issue.